webrtc.html

```diff
@@ -903,13 +901,87 @@
                     </p>
                   </li>
 
+
+                  
                   <li>
-                    <p>Set <var>connection</var>'s description attribute (
-                    <code><a>localDescription</a></code> or
-                    <code><a>remoteDescription</a></code> depending on the
-                    setting operation) to the
-                    <code><a>RTCSessionDescription</a></code> argument.</p>
-                  </li>
+
+                  <p>Set the <var>connection</var>'s varios description attributes
+                  by executing one of the following. </p>
+                  
+                  <p class="Issue"> NOTE: The principles of pending and current SDP 
+                  were agreed by the WG but the details the this step has not 
+                  yet been fully reviewed. TODO - review this. </p>
+                  
+                  <ul>
+                    
+                    <!-- A transition stable to haveLocalOffer --> <li> <p>If
+                    the local description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "offer", and it has a version that is later than the
+                    currentLocalDesc, then the pendingLocalDesc will be set to
+                    the argument and the state will transition to
+                    have-local-offer.  </p> </li>
+
+                    <!-- C transition haveRemoteOffer or haveLocalProvAnswer to
+                    stable --> <li><p>If the local description was set, and the
+                    <code><a>RTCSessionDescription</a></code> argument has a
+                    type of "answer", then this complets an offer answer
+                    negotiation and currentLocalDesc is set to the argument,
+                    currentRemoteDesc is set to the value of pendingRemoteDesc,
+                    then pendingRemoteDesc and pendingLocalDesc are set to null,
+                    and the state will transitions to stable.  </p> </li>
+
+                    <!-- D transition stable to haveRemoteOffer --> <li> <p>If
+                    the remote description was set, and the
```

Comment:
This section is under setLocalDescription.

I think the idea of having a processing model described under a method call is a bad one. We have a section (4.3.1 Operation) that has multiple processing models already (and should be clear that this is what they are). If we want a single list, I think it should be moved up there, and pointed to from both methods.